packages/leafygreen-provider/src/DarkModeContext.tsx

```diff
@@ -1,27 +1,27 @@
 import React, { createContext, PropsWithChildren, useContext } from 'react';
 import PropTypes from 'prop-types';
-import { DarkModeProps, Theme } from '@leafygreen-ui/lib';
-import type { ThemeType } from 'packages/lib/src/DarkModeProps';
 
-type DarkModeContextProps = DarkModeProps & {
-  theme?: ThemeType;
-};
+interface DarkModeContextProps {
+  globalDarkMode: boolean;
+}
 
 const DarkModeContext = createContext<DarkModeContextProps>({
-  darkMode: false,
-  theme: Theme.Light,
+  globalDarkMode: false,
 });
 export const useDarkModeContext = () => useContext(DarkModeContext);
+export const useDefaultDarkMode = (componentDarkMode?: boolean) => {
```

Comment:
I'm wondering about the name `defaultDarkMode`. It's not my favorite, but I don't know what I'd prefer